tests/testthat/test-add_significance_stars.R

```diff
@@ -89,3 +89,20 @@ test_that("add_significance_stars(pattern) messaging", {
     "no stars will be added"
   )
 })
+
+# check the stars footnote does not replace the tests footnote, issue #2184
+test_that("add_significance_stars() footnote", {
+  expect_equal(
+    trial |>
+      tbl_summary(by = trt, include = c(age, grade)) |>
+      add_p() |>
+      add_significance_stars() |>
```

Comment:
I think this does not produce the expected result:

![image](https://github.com/user-attachments/assets/878a5137-6d35-483f-b8ee-69b1353724b3)

While I expected something like the workaround:
```r
trial |>
  tbl_summary(by = trt, include = c(age, grade)) |>
  add_p() |>
  separate_p_footnotes() |>
  add_significance_stars()
  ```
  
![image](https://github.com/user-attachments/assets/19e25552-5232-4765-a172-99a2dc46d608)

Reply:
People can choose the move the footnotes to each p-value, but we don't need to force that on them.